napari/layers/points/_tests/test_points.py

```diff
@@ -158,18 +159,18 @@ def test_set_current_properties_on_empty_layer_with_color_cycle(feature_name):
 def test_empty_layer_with_text_properties():
     """Test initializing an empty layer with text defined"""
     default_properties = {'point_type': np.array([1.5], dtype=float)}
-    text_kwargs = {'text': 'point_type', 'color': 'red'}
+    text_kwargs = {'string': 'point_type', 'color': 'red'}
     layer = Points(
         property_choices=default_properties,
         text=text_kwargs,
     )
-    assert layer.text.values.size == 0
-    np.testing.assert_allclose(layer.text.color, [1, 0, 0, 1])
+    assert layer._view_text.size == 0
```

Comment:
Just looking at the changes in this test, this seems like a fairly big public API change if `layer.text.values` no longer exists. Am I getting that right? Do we need to think about deprecation here?